core/src/main/scala/cats/Eval.scala

```diff
@@ -350,7 +350,7 @@ trait EvalMonoid[A] extends Monoid[Eval[A]] with EvalSemigroup[A] {
 trait EvalGroup[A] extends Group[Eval[A]] with EvalMonoid[A] {
   implicit def algebra: Group[A]
   def inverse(lx: Eval[A]): Eval[A] =
-    lx.map(_.inverse)
+    lx.map(_.inverse())
```

Comment:
I tried removing the parens from `def inverse()` and got this:

```
[error] /Users/cody/code/cats/core/src/main/scala/cats/syntax/group.scala:15: macro implementation has incompatible shape:
[error]  required: (c: scala.reflect.macros.whitebox.Context): c.Expr[A]
[error]  or      : (c: scala.reflect.macros.whitebox.Context): c.Tree
[error]  found   : (c: reflect.macros.Context)(): c.Expr[R]
[error] number of parameter sections differ
[error]   def inverse: A = macro Ops.unop[A]
```

Paging @non, our macro optimization wizard.